### ISSUE TITLE
Bayou Playground doesnt work

### ISSUE THREAD

**Issue by kwojcicki** [capergroup/bayou#220]
When running the samples the result says "Unexpected error. Please retry.". Assuming it has to do with the errors that show up in the console 

![image](https://user-images.githubusercontent.com/2487946/62908471-3a995b00-bd46-11e9-9b24-380d9f8ee648.png)

**Comment by nnganesha** [comment]
same here... i can see a CORS error, but accessing the url directly also gives an error saying service is unavailable.

**Comment by nnganesha** [comment]
Thanks a lot for the quick response.. still seeing same error.. will await your update once it is up..

**Comment by nnganesha** [comment]
yes i tried running but seeing errors.. is there a way you can bring it up if it is down?

**Comment by nnganesha** [comment]
oh ok thanks.. so is this project still active or has been retired?
